docs/sources/getting-started/get-started-grafana-prometheus.md

```diff
@@ -112,13 +119,204 @@ remote_write:
 To configure your Prometheus instance to work with Grafana locally instead of Grafana Cloud, install Grafana [here](/grafana/download) and follow the configuration steps listed [here](/docs/grafana/latest/datasources/prometheus/#configure-the-data-source).
 {{% /admonition %}}
 
-#### Check Prometheus metrics in Grafana Explore view
+## Troubleshooting
```

Comment:
I would reformat so as to avoid stacked headings. A stacked heading is a heading immediately followed by another heading without body text between them. 

I would also be as specific as possible about the "problem", which seems to be that Prometheus isn't running as functioning as expected. Is it possible to provide more description about the user experience here? What's happening specifically that indicates to them there's a problem?

Reply:
good shout ! thanks for the feedback  @chri2547 . I just pushed a version with less headings and with descriptions that are more relatable